addons/mail/wizard/template_reset_wizard.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _
+
+
+class TemplateResetWizard(models.TransientModel):
+    _name = 'template.reset.wizard'
+    _description = 'Mail/SMS Template Reset Wizard'
+
+    def reset_template(self):
+        if self.env.context.get('active_model') and self.env.context.get('active_ids'):
```

Comment:
Would prefer to have template_id or template_ids on wizard. Dealing with context is always a bit hackish.